CameraOverlay/commons.py

```diff
@@ -1,15 +1,20 @@
 import os
 import json
+import fnmatch
 from dotenv import load_dotenv
 
-from .topics import *
-
 load_dotenv()
 CONFIG_FILE = 'configs.json'
 ACTIVE_OVERLAY_KEY = 'activeOverlay'
+OVERLAY_FILE_PATTERN = 'Overlay_Demo*.py'
 CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
 
 
+def get_overlays(dir=CURRENT_DIRECTORY):
+	overlays = [file for file in os.listdir(dir) if fnmatch.fnmatch(file, OVERLAY_FILE_PATTERN)]
```

Comment:
I once got told off at work for doing something like this as it reduced 'readability' of code. Which is more important than reducing number of lines of code. No need to change here tho, just letting you know!